rom/dev/src/main.rs

```diff
@@ -77,10 +78,8 @@ pub extern "C" fn rom_entry() -> ! {
     // Start the watchdog timer
     wdt::start_wdt(&mut env.soc_ifc);
 
-    let result = kat::execute_kat(&mut env);
-    if let Err(err) = result {
-        handle_fatal_error(err.into());
-    }
+    #[cfg(not(feature = "no-kats"))]
+    execute_kats(&mut env);
```

Comment:
Rather than using multiple #[cfg...] attributes, consider this:


```rs
    if !cfg!(feature = "no-kats") {
      let result = kat::execute_kat(&mut env);
      if let Err(err) = result {
          handle_fatal_error(err.into());
      }
    }
```

Reply:
Done